database/sql/V072__total_fee_token.sql

```diff
@@ -0,0 +1,24 @@
+ALTER TABLE order_execution
+RENAME COLUMN surplus_fee TO total_fee;
```

Comment:
new column is still called `total_fee` instead of `executed_fee` in the migration and `readme.md`.